3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/sqlquery/OutputSpec.scala

```diff
@@ -41,6 +41,6 @@ class OutputSpec extends Specification with ValidationMatchers { def is = s2"""
 
   def e4 = {
     val date = new Date(1465558727000L)
-    JsonOutput.getValue(date, "java.sql.Date") must beEqualTo(JString("2016-06-10T11:38:47.000Z"))
+    JsonOutput.getValue(date, "java.sql.Date") must beEqualTo(JString("2016-06-10T12:38:47.000+01:00"))
```

Comment:
same, it seems `toString` changed for `DateTime` between java7 and java8 regarding the tz :fearful: